block/internal/syncing/syncer.go

```diff
@@ -212,20 +216,10 @@ func (s *Syncer) processLoop() {
 	s.logger.Info().Msg("starting process loop")
 	defer s.logger.Info().Msg("process loop stopped")
 
-	blockTicker := time.NewTicker(s.config.Node.BlockTime.Duration)
-	defer blockTicker.Stop()
-
 	for {
-		// Process pending events from cache on every iteration
-		s.processPendingEvents()
```

Comment:
nit: i think this should stay here, as this is part of processing and we want to check. syncLoop just sends data to the channel.